index.html

```diff
@@ -121,7 +121,7 @@
ment&quot;},{&quot;name&quot;:&quot;listen-on&quot;,&quot;description&quot;:&quot;The target element that&apos;s listened to for context menu opening events.\nBy default the vaadin-context-menu listens to the target&apos;s `vaadin-contextmenu`\nevents.&quot;,&quot;sourceRange&quot;:{&quot;file&quot;:&quot;bower_components\\vaadin-context-menu\\vaadin-context-menu.html&quot;,&quot;start&quot;:{&quot;line&quot;:246,&quot;column&quot;:12},&quot;end&quot;:{&quot;line&quot;:251,&quot;column&quot;:13}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;HTMLElement&quot;,&quot;inheritedFrom&quot;:&quot;Vaadin.ContextMenuElement&quot;},{&quot;name&quot;:&quot;close-on&quot;,&quot;description&quot;:&quot;Event name to listen for closing the context menu.&quot;,&quot;sourceRange&quot;:{&quot;file&quot;:&quot;bower_components\\vaadin-context-menu\\vaadin-context-menu.html&quot;,&quot;start&quot;:{&quot;line&quot;:256,&quot;column&quot;:12},&quot;end&quot;:{&quot;line&quot;:260,&quot;column&quot;:13}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;string&quot;,&quot;inheritedFrom&quot;:&quot;Vaadin.ContextMenuElement&quot;},{&quot;name&quot;:&quot;no-overlap&quot;,&quot;description&quot;:&quot;If true, a dropdown content is rendered without overlapping the trigger element.&quot;,&quot;sourceRange&quot;:{&quot;start&quot;:{&quot;line&quot;:31,&quot;column&quot;:24},&quot;end&quot;:{&quot;line&quot;:34,&quot;column&quot;:25}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;boolean&quot;},{&quot;name&quot;:&quot;fit-into&quot;,&quot;description&quot;:&quot;The element to fit the dropdown content into.\nBy default, the window element is used.&quot;,&quot;sourceRange&quot;:{&quot;start&quot;:{&quot;line&quot;:40,&quot;column&quot;:24},&quot;end&quot;:{&quot;line&quot;:43,&quot;column&quot;:25}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;Object&quot;},{&quot;name&quot;:&quot;vertical-offset&quot;,&quot;description&quot;:&quot;The pixel value to be added to the dropdown vertical position.&quot;,&quot;sourceRange&quot;:{&quot;start&quot;:{&quot;line&quot;:48,&quot;column&quot;:24},&quot;end&quot;:{&quot;line&quot;:51,&quot;column&quot;:25}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;number&quot;},{&quot;name&quot;:&quot;horizontal-offset&quot;,&quot;description&quot;:&quot;The pixel value to be added to the dropdown horizontal position.&quot;,&quot;sourceRange&quot;:{&quot;start&quot;:{&quot;line&quot;:56,&quot;column&quot;:24},&quot;end&quot;:{&quot;line&quot;:59,&quot;column&quot;:25}},&quot;metadata&quot;:{},&quot;type&quot;:&quot;number&quot;}],&quot;events&quot;:[{&quot;type&quot;:&quot;CustomEvent&quot;,&quot;name&quot;:&quot;opened-changed&quot;,&quot;description&quot;:&quot;Fired when the `opened` property changes.&quot;,&quot;metadata&quot;:{},&quot;inheritedFrom&quot;:&quot;Vaadin.ContextMenuElement&quot;}],&quot;styling&quot;:{&quot;cssVariables&quot;:[],&quot;selectors&quot;:[]},&quot;slots&quot;:[],&quot;tagname&quot;:&quot;spine-context-menu&quot;}"
```

Comment:
Now iron-doc-element generates the following link of `Vaadin.ContextMenuElement`: `http://127.0.0.1:8081/mixins/Vaadin.ContextMenuElement`. This is because of internal logic of `iron-doc-element`. Please create an issue in the repository of `iron-doc-element` and create a reference to it here.